frontend/index.js

```diff
@@ -16,14 +16,31 @@ const kennedySpaceCentreTime = () => {
 kennedySpaceCentreTime();
 
 const timeZoneDifference = () => {
-  const usersTime = new Date().getTimezoneOffset() / 60;
-  const additionalTimeFromUser = 5;
+  const usersUTCOffsetHours = new Date().getTimezoneOffset() / 60;
+  const totalTimeZoneOffset = usersUTCOffsetHours + 5;
 
   const offset = document.createTextNode(
-    `${additionalTimeFromUser} hours ahead of the Kennedy Space Centre`
+    `${totalTimeZoneOffset} hours ahead of the Kennedy Space Centre`
   );
 
   document.getElementById("difference").append(offset);
 };
 
 timeZoneDifference();
+
+const solDays = document.getElementById("sol-days");
+const earthDayMs = 86400000;
+const solDayMs = 88775240;
+const convertToSol = solDayMs / earthDayMs;
+
+const calcualteSol = () => {
+  const curiosityLandingDateMs =
+    Date.now() - new Date(2012, 7, 6, 5, 17, 57).getTime();
+
+  const earthDaySinceLanding = curiosityLandingDateMs / earthDayMs;
+
+  const sol = Math.round(earthDaySinceLanding / convertToSol);
+
+  solDays.innerHTML = sol;
+};
```

Comment:
This looks good! 👍 

The only thing I'd say is to move the `solDays` and `convertToSol` variables into the function itself. The fewer variables we have that sit outside the functions, the better. This leaves only real-world constants outside the variables (`earthDayMs` and `solDayMs`)